Entities/Items/Crate/Crate.as

```diff
@@ -115,7 +118,8 @@ void onTick(CBlob@ this)
 			this.getCurrentScript().tickFrequency = 15;
 		else
 		{
-			this.getCurrentScript().tickFrequency = 0;
+			this.getCurrentScript().tickFrequency = 60;
```

Comment:
This should be 30, not 60 (not that the behavior will be different).

Reply:
Besides it should be lowered to a more sane amount (i.e. every few ticks) as to avoid (likely) expensive `getBlobsInBox` logic for each crate each tick.

Reply:
Tick frequency being a high number lowers the amount of ticking (so 60 makes it tick only every 2 seconds)

Reply:
Yep, my bad :man_facepalming: